README.md

```diff
@@ -5,7 +5,7 @@ to easily switch between different versions.
 
 ## Documentation
 
-See the [command-line reference][cli-reference] for
+See the [command-line reference](https://internetcomputer.org/docs/current/developer-docs/developer-tools/cli-tools/dfxvm/docs/cli-reference/dfxvm/dfxvm-install) for
```

Comment:
Please make this change on line 44 (this line should not change)

Reply:
Also this should link to the portal-equivalent of this page:
https://github.com/dfinity/dfxvm/blob/main/docs/cli-reference/index.md